packages/pn-personafisica-webapp/public/locales/de/notifiche.json

```diff
@@ -58,9 +58,6 @@
     "show-history": "Verlauf anzeigen",
     "show-more": "Mehr Details anzeigen",
     "show-less": "Weniger Details anzeigen",
-    "add_domicile_PEC": "PEC hinzufügen",
-    "add_domicile_EMAIL": "E-Mail-Adresse hinzufügen",
-    "add_domicile_APPIO": "Dienst aktivieren",
     "domicile_PEC": "Hast du eine PEC-Adresse? Füge sie hinzu und du bekommst keine Einschreiben mehr in Papierform.",
```

Comment:
remove domicile_PEC, domicile_EMAIL and domicile_APPIO here and in all localization files